ML-based row-level anomaly detection (#990)
databrickslabs/dqx: docs/dqx/docs/guide/anomaly_detection.mdx

---
sidebar_position: 15
---

# Anomaly Detection (ML-based)

> Automatically detect unusual data patterns using machine learning. DQX trains models on your "normal" data patterns, then flags suspicious records in production pipelines.

Reviewer: ```suggestion
Automatically detect unusual data patterns using machine learning. DQX trains models on your "normal" data patterns, then flags records that deviate.
```
It's about data, not pipelines. It may not be production.
